### ISSUE TITLE
Can't build and run master branch

### ISSUE THREAD

**Issue by bren3582** [voisine/breadwallet#374]
First of all, I want to say thank you Aaron Voisine and other contributors for the excellent iOS wallet. I use it all the time and it's great to see how active development has been.

I've created my own fork of the master branch to implement new features for my own use (and possibly to contribute upstream if they are approved), but I'm unable to build the project due to multiple roadblocks.

First of all, breadwallet-Swift.h doesn't seem to exist. I "fixed" this by commenting lines 33, 46, 59-66, 320-324, and 496 in BRSettingsViewController.m. Additionally, I ran into issues with WatchApp, I believe because I don't have the App Groups entitlement on my Apple ID (not a paid developer account). To solve this, I turned off App Groups under the project Capabilities section then deleted WatchApp from the Target Dependencies and Embed Watch Content sections under Build Phases. Finally, I used my own bundle identifiers for the breadwallet and TodayExtension targets, which is expected.

I hope that you can find some time to take a look at these issues. I think that breadwallet being open source should mean that users and developers alike can run the code as is on their own devices without jumping through hoops and disabling functionality. I'm still not sure how the changes I had to make will affect the app at runtime.

Thanks,
Brian

**Comment by voisine** [comment]
Thanks @bren3582, we were actually just discussing how to make the project easier to build for those who don't have developer accounts now that Xcode 7 allows you to do this with just an iTunes account.

I'd like to get to where it's just a one click build. Specifically breadwallet-Swift.h is supposed to be auto generated. I assume you are using the latest Xcode 7.3?

I need to investigate the app group issue, perhaps we will have to have it off by default if it's not available for free-developer accounts, and then I think we can change the bundle identifiers to contain a variable name that will be different for each developer.

**Comment by bren3582** [comment]
Thanks for the quick reply! That's great news that you've had this issue in mind already. Yes, I am using the latest Xcode 7.3 so I'm not sure what the issue is with breadwallet-Swift.h. I'll try to investigate further.

I was able to get the WatchApp to build as well by tinkering more with bundle identifiers in various Info.plist files. It seems the App Group was the main issue. Rather than try to list everything I changed, I've made a commit to my fork that includes all of the changes.
